src/components/Dropdown.js

```diff
@@ -32,7 +32,7 @@ const Dropdown = React.createClass({
   },
 
   renderButton() {
-    return dom.span(
+    return dom.button(
```

Comment:
This adds some funny styles

You should be able to reproduce by adding some more editor tabs an observing the hidden tabs dropdown appear.

![screen shot 2016-11-26 at 8 27 05 pm](https://cloud.githubusercontent.com/assets/254562/20644914/d26f6712-b416-11e6-940a-3b9fe3f36ac5.png)